test/spec/runners/ssz_static.ex

```diff
@@ -40,47 +78,57 @@ defmodule SszStaticTestRunner do
     expected =
       YamlElixir.read_from_file!(case_dir <> "/value.yaml")
       |> SpecTestUtils.sanitize_yaml()
+      |> sanitize(schema)
```

Comment:
Let's move both the `sanitize` function and the `parse_type` to a spec test utils. We shouldn't be duplicating this if we already have it in the `ssz_generic` tests.